odoo/tests/common.py

```diff
@@ -471,18 +471,19 @@ def get_unaccent_wrapper(cr):
                     self.env.user.flush()
                     self.env.cr.flush()
 
-        self.assertEqual(
-            len(actual_queries), len(expected),
-            "\n---- actual queries:\n%s\n---- expected queries:\n%s" % (
-                "\n".join(actual_queries), "\n".join(expected),
-            )
-        )
-        for actual_query, expect_query in zip(actual_queries, expected):
+        if self.warm:
```

Comment:
I suggest to use some early return instead, as it makes the diff really simple
```suggestion
        if not self.warm:
            return
```